README.md

```diff
@@ -7,17 +7,44 @@ common to multiple components.
 For details of the other Kata Containers repositories, see the
 [repository summary](https://github.com/kata-containers/kata-containers).
 
-* [Kata Containers Architecture](architecture.md)
-* [Developer Guide](Developer-Guide.md)
-* [Code of Conduct](CODE_OF_CONDUCT.md)
-* [Contributing](CONTRIBUTING.md)
-* [Kata Containers design](./design/README.md)
-* [Documentation Requirements](Documentation-Requirements.md)
-* [Installation guides](./install/README.md)
-* [Kata Containers with k8s and cri-containerd](./how-to/how-to-use-k8s-with-cri-containerd-and-kata.md)
-* [Licensing](Licensing-strategy.md)
-* [Limitations](Limitations.md)
+## Getting Started
+
+* [Installation guides](./install/README.md): Install and run Kata Containers with Docker or Kubernetes
+
+## More User Guides
+
+* [Upgrading](Upgrading.md): how to upgrade from [Clear Containers](https://github.com/clearcontainers) and [runV](https://github.com/hyperhq/runv) to [Kata Containers](https://github.com/kata-containers) and how to upgrade an existing Kata Containers system to the latest version.
+* [Limitations](Limitations.md): differences and limitations compared with the default [Docker](https://www.docker.com/) runtime,
+[`runc`](https://github.com/opencontainers/runc).
+
+* HOWTO: [Kata Containers with k8s and cri-containerd](./how-to/how-to-use-k8s-with-cri-containerd-and-kata.md)
+* HOWTO: [OpenStack Zun with Kata Containers](zun/zun_kata.md)
+
+## Developer Guide
+
+Documents that help to understand and contribute to Kata Containers.
+
+### Design and Implementations
+
+* [Kata Containers Architecture](architecture.md): Architectural overview of Kata Containers
+* [Kata Containers design](./design/README.md): More Kata Containers design documentations
+
+### How to Contribute
+
+* [Developer Guide](Developer-Guide.md): Setup the Kata Containers developing environments
+* [How to contribute to Kata Containers](https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md)
+
+### Code Licensing
+
+* [Licensing](Licensing-strategy.md): About the licensing strategy of Kata Containers.
+
+### The Release Process
+
 * [Release strategy](Releases.md)
 * [Release Checklist](Release-Checklist.md)
-* [Upgrading](Upgrading.md)
-* [OpenStack Zun with Kata Containers](zun/zun_kata.md)
+
+## Help Improving the Documents
+
+* [Documentation Requirements](Documentation-Requirements.md)
+* [Contributing](CONTRIBUTING.md)
```

Comment:
Maybe the Contributing and CoC entries should move up to the Contributing section?

Reply:
oh, they don't 301 for me - if you look on the 'files changed' tab of this PR, and then open the 'view' button (which views your modified file in your branch), then those contrib and CoC links work fine for me :-)

Sure, drop the extra CONTRIBUTING. I think I'd still move the CoC up into the contributing section.

Reply:
OK for me. They are a bit like when I unboxed a gift package and got a smaller package 😄